admin/cluster_setup.py

```diff
@@ -143,11 +143,6 @@ def main(reactor, args, base_path, top_level):
                                )
         gather_deferreds(results)
 
-        if options['apps-per-node'] > 0:
```

Comment:
Remove the `apps-per-node` and `app-template` options from `RunOptions` and all the code after `main`, which is no longer used.